utils.py

```diff
@@ -267,21 +268,43 @@ def get_random_choice(alist):
     return alist[index]
 
 
+def convert_png_data_url_to_binary(content):
+    """Converts a PNG base64 data URL (represented by 'content') to a PNG
+    binary data.
+
+    Args:
+        content: str. PNG data URL.
```

Comment:
A string that is to be interpreted as a PNG data URL.

Reply:
Done.